tools/cldr/cldr-to-icu/README.md

```diff
@@ -103,11 +103,53 @@ $ ant -f build-icu-data.xml
+  +            <altPath target="//ldml/dates/calendars/calendar[@type='gregorian']/timeFormats/timeFormatLength[@type='long']/timeFormat[@type='standard']/pattern[@type='standard']"
+  +                     source="//ldml/dates/calendars/calendar[@type='gregorian']/timeFormats/timeFormatLength[@type='long']/timeFormat[@type='standard']/pattern[@alt='ascii'][@type='standard']"
+  +                     locales="en"/>
+  +            <altPath target="//ldml/dates/calendars/calendar[@type='gregorian']/timeFormats/timeFormatLength[@type='medium']/timeFormat[@type='standard']/pattern[@type='standard']"
+  +                     source="//ldml/dates/calendars/calendar[@type='gregorian']/timeFormats/timeFormatLength[@type='medium']/timeFormat[@type='standard']/pattern[@alt='ascii'][@type='standard']"
+  +                     locales="en"/>
+  +            <altPath target="//ldml/dates/calendars/calendar[@type='gregorian']/timeFormats/timeFormatLength[@type='short']/timeFormat[@type='standard']/pattern[@type='standard']"
+  +                     source="//ldml/dates/calendars/calendar[@type='gregorian']/timeFormats/timeFormatLength[@type='short']/timeFormat[@type='standard']/pattern[@alt='ascii'][@type='standard']"
+  +                     locales="en"/>
+  +            <altPath target="//ldml/dates/calendars/calendar[@type='generic']/dateTimeFormats/availableFormats/dateFormatItem[@id='Ehm']"
+  +                     source="//ldml/dates/calendars/calendar[@type='generic']/dateTimeFormats/availableFormats/dateFormatItem[@id='Ehm'][@alt='ascii']"
+  +                     locales="en"/>
+  +            <altPath target="//ldml/dates/calendars/calendar[@type='generic']/dateTimeFormats/availableFormats/dateFormatItem[@id='Ehms']"
+  +                     source="//ldml/dates/calendars/calendar[@type='generic']/dateTimeFormats/availableFormats/dateFormatItem[@id='Ehms'][@alt='ascii']"
+  +                     locales="en"/>
+  +            <altPath target="//ldml/dates/calendars/calendar[@type='generic']/dateTimeFormats/availableFormats/dateFormatItem[@id='h']"
+  +                     source="//ldml/dates/calendars/calendar[@type='generic']/dateTimeFormats/availableFormats/dateFormatItem[@id='h'][@alt='ascii']"
+  +                     locales="en"/>
+  +            <altPath target="//ldml/dates/calendars/calendar[@type='generic']/dateTimeFormats/availableFormats/dateFormatItem[@id='hm']"
+  +                     source="//ldml/dates/calendars/calendar[@type='generic']/dateTimeFormats/availableFormats/dateFormatItem[@id='hm'][@alt='ascii']"
+  +                     locales="en"/>
+  +            <altPath target="//ldml/dates/calendars/calendar[@type='generic']/dateTimeFormats/availableFormats/dateFormatItem[@id='hms']"
+  +                     source="//ldml/dates/calendars/calendar[@type='generic']/dateTimeFormats/availableFormats/dateFormatItem[@id='hms'][@alt='ascii']"
   +                     locales="en"/>
   ```
   Then run the generator:
```

Comment:
I used them to generate the .txt file to see what we get.

They work, and there is no U+202F left in the Gregorian calendar.
Except...
We also have it in some `intervalFormats`:
```xml
<intervalFormats>
    <intervalFormatItem id="h">
        <greatestDifference id="a">h a – h a</greatestDifference>
        <greatestDifference id="h">h – h a</greatestDifference>
    </intervalFormatItem>
    <intervalFormatItem id="hm">
        <greatestDifference id="a">h:mm a – h:mm a</greatestDifference>
        <greatestDifference id="h">h:mm – h:mm a</greatestDifference>
        <greatestDifference id="m">h:mm – h:mm a</greatestDifference>
    </intervalFormatItem>
    <intervalFormatItem id="hmv">
        <greatestDifference id="a">h:mm a – h:mm a v</greatestDifference>
        <greatestDifference id="h">h:mm – h:mm a v</greatestDifference>
        <greatestDifference id="m">h:mm – h:mm a v</greatestDifference>
    </intervalFormatItem>
    <intervalFormatItem id="hv">
        <greatestDifference id="a">h a – h a v</greatestDifference>
        <greatestDifference id="h">h – h a v</greatestDifference>
    </intervalFormatItem>
```

Trouble is, they don't have `alt="ascii"` variants.
So we can't just add rules, we need to add new entries in the xml.

Reply:
I assume that CLDR didn't add alt="ascii" for interval formats because they were less disruptive.
Thanks for testing!

Reply:
Yes, people are not parsing the interval formats to extract a time, so we intentionally did not add the alt="ascii" variant for those.